part-spec/definitions.json

```diff
@@ -429,13 +456,10 @@
           "description": "maximum unit quantity corresponding to unit text - example 40mV would have a value of 40",
           "type": "number"
         },
-        "unitText": {
-          "description": "human readable text describing value - example 40mV would have a value of mV",
-          "type": "string"
-        },
         "unitFactor": {
-          "description": "multiplier on the value to achieve the SI unit - example for 40mV the unitFactor would be 0.001",
-          "type": "number"
+          "description": "multiplier on the value to achieve the SI unit selected - example for 40mV the unitFactor would be 0.001",
```

Comment:
Do we still need this or can we let the tools add this intelligence?

Reply:
I think Herman still wanted the ability to do this.